tracking/old_daemon_tracking.py

```
@@ -0,0 +1,144 @@
+"""
+Script to download and track UPs and Zonas de Regulación (designed to be run as a cron job) 
+"""
+import os
+from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)